app/src/main/res/layout/activity_custom_selector.xml

```diff
@@ -14,6 +14,52 @@
       android:layout_width="match_parent"
       android:layout_height="0dp"
       app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintBottom_toTopOf="@id/bottom_layout"
       app:layout_constraintTop_toBottomOf="@+id/toolbar_layout"/>
 
+    <androidx.constraintlayout.widget.ConstraintLayout
+      android:id="@+id/bottom_layout"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:background="@color/white"
+      android:visibility="gone"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent">
+
+        <Button
+          android:id="@+id/not_for_upload"
+          android:layout_width="0dp"
+          android:layout_height="wrap_content"
+          android:text="Mark as not for upload"
```

Comment:
Similarly for the text, you can use @string/<name of your variable>, 
This is very important so as to make the application multilingual. If you use English here, the text will always be in English. But if you declare them as string constants, the value would be translated by the translated wiki to different languages. Thus when a user changes the app language, all the text in the application would be derived from the specific language strings.

Reply:
As we are in an early phase, doing quick prototyping without proper internationalization is OK as long as you don't forget it later, especially for things that might be removed/modified later.
I believe this text has high chances of making it into master, though :-)